Woodog/templates/woodogdata/sign-up.html

```diff
@@ -0,0 +1,123 @@
+
```

Comment:
Good Job :star_struck: @coding-geek21, but I'd like to request few changes,

* You could have edited the `templates/authentication/register.html` instead of creating a new template in `/woodogdata/`.
* Please add `base.html` in `templates/partials` directory. By doing that, we can make the base template available in all other template files.